src/textual/widgets/_digits.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+from typing import cast
+
+from rich.align import Align, AlignMethod
+from rich.console import RenderableType
+
+from ..geometry import Size
+from ..renderables.digits import Digits as DigitsRenderable
+from ..widget import Widget
+
+
+class Digits(Widget):
+    """A widget to display numerical values using a 3x3 grid of unicode characters.
+
+    Args:
```

Comment:
Correct me if I'm wrong, but I thought we'd settled on the convention that `__init__` args would be documented in the `__init__` docstring.

Reply:
Did we? You may be right.